### PR TITLE
UICAL-206 test AllCalendarView

### PR DESCRIPTION
**Pull request by zburke**
Test `./src/views/AllCalendarView`.

* Elaborate stripes-core mock copied mostly verbatim from ui-users, minus a few things TS barfed on.
* `withHistoryConfiguration` mirrors `withIntlConfiguration` but provides a react-router provider.
* Minor bug fix: localize the "All calendars" pane title

Refs [UICAL-206](https://issues.folio.org/browse/UICAL-206)

### CODE REVIEW

**Comment by zburke** [comment]
Oh dang. I'm guessing the stripes-core mock is breaking other tests. You can hold off on reviewing while I investigate.

File: src/test/__mocks__/stripes-core.mock.ts
Comment (ncovercash) [inline src/test/__mocks__/stripes-core.mock.ts:24]:
```suggestion
    useStripes: () => STRIPES,
    useOkapiKy: jest.fn(),
```

File: src/views/AllCalendarView.tsx
Comment (ncovercash) [inline src/views/AllCalendarView.tsx:42]: Good catch, thanks!

**Review by ncovercash — APPROVED** [review]

File: src/test/__mocks__/stripes-core.mock.ts
Comment (ncovercash) [reply src/test/__mocks__/stripes-core.mock.ts:24]: Oops, didn't realize this was pending as part of a review and wasn't commented earlier; disregard.

**Comment by ncovercash** [comment]
Good to merge as soon as sonar approves! 🙌

**Pull request merged**
